src/command_controller.rs

```diff
@@ -196,23 +263,29 @@ where
         &self,
         context: ReconciliationContext<Self::Item>,
     ) -> Result<Self::State, Error> {
-        Ok(CommandState { context })
+        Ok(CommandState {
+            context,
+            owner: None,
+            owner_reference: None,
+        })
     }
 }
 
-/// This creates an instance of a [`Controller`] which waits for incoming events and reconciles them.
+/// This creates an instance of a [`Controller`] which waits for incoming commands.
+/// For each command, we set the owner reference of our main controller custom resource, to
```

Comment:
I know what you want to say here but I don't understand this sentence.

Something along these lines maybe?
```suggestion
/// For each command, we try to find the referenced resource and will set the Owner Reference of the command to this referenced resource.
/// If we can't find the referenced object we TODO <- Malte, what do we do then?.
/// This means that the controller of the parent resource can now watch for commands and this helper controller will make sure that they trigger a reconcile for the parent by setting the OwnerReference.
```